pkg/peerhealth/peerhealth.pb.go

```diff
@@ -7,10 +7,11 @@
 package peerhealth
 
 import (
-	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
-	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
 	reflect "reflect"
 	sync "sync"
+
```

Comment:
This is very probably caused by an old `sort-import` binary on your machine, and since CI doesn't check for modified files (they are modified by the `generate` target), it isn't catched. Same for the other file. Why is no one else wondering about this unrelated change? 🤔

Reply:
Cool thanks, I thought it's the other way around (i.e that the change was due to the new sort-import version).

> Why is no one else wondering about this unrelated change? 🤔

Probably others assumed the same as myself - I assume that the commit name contributed to that notion as well :sweat_smile: